UDPClientGUI/SiceUDPClient.cs

```diff
@@ -51,5 +60,32 @@ public void Dispose()
         {
             client.Dispose();
         }
+
+        public void Handle(ClientCommandEvent Command)
+        {
+            Dictionary<ClientCommandEvent.Command, System.Action> handler = new Dictionary<ClientCommandEvent.Command, System.Action>();
+            handler.Add(ClientCommandEvent.Command.Connect, Connect);
+            handler.Add(ClientCommandEvent.Command.Disconnect, Disconnect);
+            handler.Add(ClientCommandEvent.Command.SendMessage, SendMessage);
+            InputMessage = Command.Message;
+            ConnectionIP = Command.ConnectionIP;
+            ConnectionPort = Command.ConnectionPort;
```

Comment:
This should be in the constructor, otherwise there's no advantage in having it.

Reply:
Moved to constructor